src/DinfTransLimAccum.cpp

```diff
@@ -244,7 +245,11 @@ int tlaccum(char *angfile, char *tsupfile, char *tcfile, char *tlafile, char *de
 							if(usec==1)
 							{
 								if(csout->isNodata(in,jn))con=true;
-								else loadin=loadin+p*tempFloat*csout->getData(in,jn,tempFloat);
+								
```

Comment:
This code is puzzling.  Is the intent to square the variable being read from csout.  If so this is correct.  If not it is incorrect.  In any event it is not very readible.